shell/bwrap.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -ue
```

Comment:
Do you definitely need bash here?

Reply:
well, pure `sh` doesn't really support arrays, and properly escaping a list of arguments is tedious without them...

Reply:
Makes sense - couldn't find an easy way in /bin/sh indeed